paparazzi/src/test/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtensionTest.kt

```diff
@@ -77,7 +77,7 @@ class AccessibilityRenderExtensionTest {
             relativePath = expected.path,
             image = image,
             goldenImage = ImageIO.read(expected),
-            maxPercentDifferent = 0.1, // Default percent used in Paparazzi constructor.
+            maxPercentDifferent = 0.2,
```

Comment:
odd, why does this show up in the diff?  this already landed on master...

Reply:
perhaps, stale master on your fork?

Reply:
Just a guess that I don't fully understand either: you added a merge commit 2b4509a to master together with ba3ced7 commit in #315. But this PR only has the ba3ced7 commit, so it's not a clean linear history.